<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppliGui</class>
 <widget class="QMainWindow" name="AppliGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_CurrentProfil">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string>- aucun profil  selectioné -</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_creerProfil">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Créer un profil</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_afficheProfil">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Afficher données publiques</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_modifProfil">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Modifier les données</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_affiche_privateData">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Afficher données privées</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu_principal">
    <property name="title">
     <string>menu principal</string>
    </property>
    <widget class="QMenu" name="menumessage">
     <property name="title">
      <string>message</string>
     </property>
     <addaction name="actionEcrire_un_message"/>
     <addaction name="actionLire_les_messages"/>
    </widget>
    <addaction name="menumessage"/>
    <addaction name="actionordonnance"/>
    <addaction name="actionimprimer"/>
    <addaction name="separator"/>
    <addaction name="actionfermer"/>
   </widget>
   <widget class="QMenu" name="menupi_ces_jointes">
    <property name="title">
     <string>pièces jointes</string>
    </property>
    <widget class="QMenu" name="menuajouter">
     <property name="title">
      <string>ajouter</string>
     </property>
     <addaction name="actionOrdonance"/>
     <addaction name="actionBilan_sanguin"/>
     <addaction name="actionImagerie"/>
     <addaction name="actionPi_ces_personnelles"/>
    </widget>
    <addaction name="menuajouter"/>
    <addaction name="actionconsulter"/>
   </widget>
   <widget class="QMenu" name="menuinfos_pratiques">
    <property name="title">
     <string>infos pratiques</string>
    </property>
    <addaction name="actionlire_infos"/>
    <addaction name="separator"/>
    <addaction name="action_propos"/>
   </widget>
   <widget class="QMenu" name="menucharger_un_profil">
    <property name="title">
     <string>charger un profil</string>
    </property>
   </widget>
   <widget class="QMenu" name="menugestionnaire">
    <property name="title">
     <string>gestionnaire</string>
    </property>
    <widget class="QMenu" name="menuchanger_de_langue">
     <property name="title">
      <string>changer de langue</string>
     </property>
     <addaction name="actionFran_ais"/>
     <addaction name="actionAnglais"/>
    </widget>
    <addaction name="menuchanger_de_langue"/>
    <addaction name="actionsupprimer_un_profil"/>
    <addaction name="actionsauvergarder_vos_donn_es"/>
    <addaction name="actioncharger_vos_donn_es_sauvegard_es"/>
    <addaction name="actionmot_de_passe_oubli"/>
   </widget>
   <addaction name="menumenu_principal"/>
   <addaction name="menupi_ces_jointes"/>
   <addaction name="menucharger_un_profil"/>
   <addaction name="menugestionnaire"/>
   <addaction name="menuinfos_pratiques"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionordonnance">
   <property name="text">
    <string>ordonnance</string>
   </property>
  </action>
  <action name="actionimprimer">
   <property name="text">
    <string>imprimer</string>
   </property>
  </action>
  <action name="actionfermer">
   <property name="text">
    <string>fermer</string>
   </property>
  </action>
  <action name="actionconsulter">
   <property name="text">
    <string>consulter</string>
   </property>
  </action>
  <action name="actionlire_infos">
   <property name="text">
    <string>lire infos</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>à propos</string>
   </property>
  </action>
  <action name="actionsupprimer_un_profil">
   <property name="text">
    <string>supprimer un profil</string>
   </property>
  </action>
  <action name="actionsauvergarder_vos_donn_es">
   <property name="text">
    <string>sauvergarder vos données</string>
   </property>
  </action>
  <action name="actionFran_ais">
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="text">
    <string>Anglais</string>
   </property>
  </action>
  <action name="actionmot_de_passe_oubli">
   <property name="text">
    <string>mot de passe oublié</string>
   </property>
  </action>
  <action name="actionEcrire_un_message">
   <property name="text">
    <string>Ecrire un message</string>
   </property>
  </action>
  <action name="actionLire_les_messages">
   <property name="text">
    <string>Lire les messages</string>
   </property>
  </action>
  <action name="actionLire_les_5_derniers_messages">
   <property name="text">
    <string>Lire les 5 derniers messages</string>
   </property>
  </action>
  <action name="actioncharger_vos_donn_es_sauvegard_es">
   <property name="text">
    <string>charger vos données sauvegardées</string>
   </property>
  </action>
  <action name="actionOrdonance">
   <property name="text">
    <string>Ordonance</string>
   </property>
  </action>
  <action name="actionBilan_sanguin">
   <property name="text">
    <string>Bilan sanguin</string>
   </property>
  </action>
  <action name="actionImagerie">
   <property name="text">
    <string>Imagerie</string>
   </property>
  </action>
  <action name="actionPi_ces_personnelles">
   <property name="text">
    <string>Pièces personnelles</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
